<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Spandana-PC" failures="1" tests="2" name="Test(failed)(failed)" time="0.061" errors="0" timestamp="21 Dec 2018 13:49:08 GMT">
  <testcase classname="test.practice.Test_Failed_Test" name="test4" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.practice.Test_Failed_Test.test4(Test_Failed_Test.java:29)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase classname="test.practice.Test_Failed_Test" name="test2" time="0.003"/>
  <testcase classname="test.practice.Test_Failed_Test" name="test3" time="0.0">
    <ignored/>
  </testcase> <!-- test3 -->
  <testcase classname="test.practice.Test_Failed_Test" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Test(failed)(failed) -->
